oauth2_http/java/com/google/auth/oauth2/UserCredentials.java

```diff
@@ -138,6 +138,13 @@ static UserCredentials fromJson(Map<String, Object> json, HttpTransportFactory t
   /**
    * Returns credentials defined by a JSON file stream using the format supported by the Cloud SDK.
    *
+   * <p>Important: If you accept a credential configuration (credential JSON/File/Stream) from an
```

Comment:
no need to add this for UserCredentials and ServiceAccounJwtAccessCredentials. For context, see [comment](http://b/389127605#comment10)